File: struts/WebContent/1019/person.js
//person.js

/*
<클래스 정의>
클래스명 = function(){

}

<속성도 정의>
클래스명=function(){
	this.속성명=데이터;
}

<함수(메소드) 정의>
클래스명.prototype.함수명=function(){

}
*/

/*Person = function(){
	alert('Person생성자');
}*/

Person = function(name,age,job){
	this.name=name;
	this.age=age;
	this.job=job;
}

Person.prototype.toString=function(){
	return "[이름:"+this.name+", 나이:"+this.age+", 직업:"+this.job+"]";
}

Person.prototype.setAge=function(age){
	this.age=age;
}

Person.prototype.setJob=function(job){
	this.job=job;
}

Person.prototype.getName=function(){
	return this.name;
}

File: JavaApp/src/com/kosta/_0829/Person.java
package com.kosta._0829;

public class Person {
    private String name;
    private String job;
    private String email;
    
	public Person(String name, String job, String email) {
		this.name = name;
		this.job = job;
		this.email = email;
	}
    
	@Override
	public String toString() {
		return "[이름:"+name+", 직업:"+job+", 이메일:"+email+"]";
	}
    
}
